data-table/table.js

```diff
@@ -515,7 +514,7 @@ function deleteSingleRow(config, rowIndex) {
  * @returns {string}            - Returns a magic string unique to a cell, based on location
  */
 function cellIndexToElementId(wrapperDivId, rowIndex, colIndex) {
-    return '_' + wrapperDivId + "_row_" + rowIndex + "_and_col_" + colIndex + '_';
+    return wrapperDivId + "_row_" + rowIndex + "_and_col_" + colIndex;
```

Comment:
I think this is a merge conflict failure? / I think everything is supposed to end in `_` now.

Reply:
Ah, yeah, you're totally right - my fault. Fixed.